gui/window_character.gui

```diff
@@ -726,7 +726,9 @@ window = {
 								tooltip = "[Character.GetUINameNotMeNoTooltip]"
```

Comment:
You can find out the age in the character finder.
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>

Reply:
> You can find out the age in the character finder.
> Click to expand

And also the Character select screen